samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/WCSLayers.razor

```diff
@@ -0,0 +1,124 @@
+
+    public async Task AddRemoveWCSLayer()
+    {
+        WCSLayer wcsSampleLayer = new()
+            {
+                Url = _wcsLayerUrl,
+                Renderer = CreateRenderer(),
+                MultidimensionalDefinition = CreateDimensionalDefinition(),
+                Opacity = 0.5,
+                Title = "Sea temperature - sea level"
+            };
+
+        if (!_view.Map.Layers.Any())
+        {
+            await _view.AddLayer(wcsSampleLayer);
+
+        }
+        else
+        {
+            _view.Map.Layers.Clear();
+        }
+        _view.Refresh();
+
+    }
+
+    public List<DimensionalDefinition> CreateDimensionalDefinition()
+    {
+        // initializes the dimensional filter for the raster display
+        List<DimensionalDefinition>? multidimensionalDefinition = new()
+            {
+                new()
+                {
+                    VariableName = "water_temp", // water temp at sea level
+                    DimensionName = "StdZ",
+                    Values = new List<int> { 0 }
+                }
+
+            };
+        return multidimensionalDefinition;
+    }
+
+
+    public RasterStretchRenderer CreateRenderer()
+    {
+        // This initializes the renderer as a standard deviation type
+        var displayRenderer = new RasterStretchRenderer
+            {
+                
+                StretchType = StretchType.StandardDeviation,
+                Statistics = new List<List<int>>{ new List<int> { -3, 37, 1, 1 } },
+                NumberOfStandardDeviations = 3
+            };
+
+        var toFromColor1 = new AlgorithmicColorRamp
+            {
+                FromColor = new MapColor( 0, 0, 255 ),
+                ToColor = new MapColor ( 0, 255, 255 )
+            };
+
+        var toFromColor2 = new AlgorithmicColorRamp
+            {
+                FromColor = new MapColor ( 0, 255, 255 ),
+                ToColor = new MapColor ( 255, 255, 0 )
+            };
+
+        var toFromColor3 = new AlgorithmicColorRamp
+            {
+                FromColor = new MapColor ( 255, 255, 0 ),
+                ToColor = new MapColor ( 255, 0, 0 )
+            };
+
+
+        // Builds the multipart color ramp
+        var colorRamp = new MultipartColorRamp
+            {
+                ColorRamps = new AlgorithmicColorRamp[]
+                {
+                    toFromColor1,
+                    toFromColor2,
+                    toFromColor3
+                }
+            };
+
+        displayRenderer.ColorRamp = colorRamp;
+        return displayRenderer;
+    }
+
+    private AlgorithmicColorRamp[]? _colorRampsList;
+    private MultipartColorRamp? multipartColorRamp;
+    private MapView _view;
+    private bool _markup = false;
+    private bool _visible = false;
+    private bool _mapRendered = false;
+
+    private string _wcsLayerUrl2 = "https://sampleserver6.arcgisonline.com/arcgis/services/ScientificData/SeaTemperature/ImageServer/WCSServer";
```

Comment:
This variable is never used and can be deleted.

Reply:
an oversight while I was cleaning up the file...corrected.